src/lib/getCurrencyOptions.tsx

```diff
@@ -3,14 +3,17 @@ import { Currency } from 'src/graphql/types.generated';
 
 export const getPledgeCurrencyOptions = (
   pledgeCurrencies: Currency[] | undefined | null,
+  format = 'long',
```

Comment:
To make this function safer, we could make format into a enum, to prevent values which aren't allowed being passed in.

Reply:
From memory it could look like this.

export enum pleadgeCurrencyOptionFormatEnum = {
  Long = 'long,
  Short = 'short'
  ...other formats
}


```suggestion
  format = pleadgeCurrencyOptionFormatEnum.long,
```

Reply:
I think this will work, but you might need to create a type for the function if it doesn't work. It should though.